content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+how you practice [continuous delivery using Pulumi]({{< relref "/docs/guides/continuous-delivery" >}}), you’ll now see
+links to things like the [Circle CI job](http://circleci.com) or [Travis CI build](http://travis-ci.com) as appropriate.
+
+![Link to the Travis CI Build from the Pulumi Console](./deep-linking-cicd-providers.png)
+
+> Pulumi supports a variety of CI/CD providers, but if yours isn't listed in [our CI/CD guide]({{< relref "/docs/guides/continuous-delivery" >}})
+> [let us know](https://slack.pulumi.com) or [contribute it](https://github.com/pulumi/pulumi/tree/master/pkg/util/ciutil)
+> on your own.
+
+## Pretty Printing JSON / Multi Configuration
+
+We now pretty print configuration values that look like JSON, and have much better support for viewing multi-line data.
+
+![JSON and mutiline configuration values](./pretty-print-multiline-json.gif)
+
+## Download Earlier Checkpoints
+
+The most important job of the Pulumi Console is to maintain a durable, accurate snapshot of your cloud resource data.
+While in most cases your [stack's checkpoint data]({{< relref "/docs/intro/concepts/state" >}}) is a low-level detail
+you don't need to worry about, in some advanced scenarios you may need to inspect or edit it manually.
+
+You can now allow download a stack’s checkpoint file directly from the Pulumi Console. (Though you can sill access the
+same data from the command-line, using `pulumi stack export` ([documentation]({{< relref "/docs/reference/cli/pulumi_stack_export" >}})),
+which now supports a `--version` flag to export older checkpoint files too.
+
+![Download stack checkpoints from the Pulumi Console](./download-checkpoint-file.png)
+
+## Reverse Stack Permissions View
+
+The Team Pro and Enterprise editions support [role-based access control]({{< relref "/docs/intro/console/collaboration/stack-permissions" >}})
+using [teams]({{< relref "/docs/intro/console/collaboration/teams" >}}). But a common problem we've heard from people in large organizations
+is that it can be difficult to review exactly *_what_* access someone has access to a stack and *_why_*.
+
+That's why the recent feature I'm personally the most excited about, is the ability for organization administrators to drill into their
+members and review stack permissions.
+
+You can see what stacks a member has access to.
+
+![Stacks an organization member has access to](./list-of-accessible-stacks.png)
+
+And also see what sources are granting that stack access.
+
+![Sources granting stack permissions](./stack-access-grants.png)
+
+## Log Rendering Performance
+
+Sadly, there isn't a useful screen shot to go along with this improvement. But we've made some changes that dramatically
+improve the Pulumi Console's performance when rendering update logs for large stacks.
+
+## Audit Logs
+
+Users of our Enterprise tier typically have a lot of stacks and teams. This can make it difficult to review what's currently in-motion
```

Comment:
```suggestion
Users of Pulumi Enterprise typically have a lot of stacks and teams. This can make it difficult to review what's currently in-motion
```